store/src/main.rs

```diff
@@ -1,25 +1,27 @@
 pub mod cli;
 pub mod config;
+pub mod db;
+pub mod migrations;
 pub mod server;
-use miden_node_utils::Config;
-
+pub mod types;
+use anyhow::Result;
 use clap::Parser;
 use cli::{Cli, Command};
```

Comment:
Do we need to use `pub mod` here or would just mod` work?

Reply:
removed `pub`